CaloOnlineTools/HcalOnlineDb/test/genLUT.sh

```diff
@@ -1,10 +1,27 @@
 #!/bin/bash
 
+dumpHelpAndExit(){
+cat << EOF
+
+examples:
+    ./genLUT.sh dumpAll card=cardPhysics.sh
```

Comment:
Please add a sample `cardPhysics.sh` so that a future user of this code does not have to reverse-engineer the syntax.

Reply:
added a skeleton file